src/SpotClient.ts

```diff
@@ -1689,4 +1925,340 @@ export class SpotClient extends BaseRestClient {
+   *
+   * REST - FUNDING -Deposit
+   * DEPRECATED
+   */
+
+  /**
+   * @deprecated This method is deprecated. Please use createDepositAddressV3() instead.
+   */
+  createDepositAddress(params: {
+    currency: string;
+    chain?: string;
+  }): Promise<APISuccessResponse<DepositAddress>> {
+    return this.postPrivate('api/v1/deposit-addresses', params);
+  }
+
+  /**
+   * @deprecated This method is deprecated. Please use getDepositAddressesV3() instead.
+   */
+  getDepositAddressesV2(params: {
+    currency: string;
+  }): Promise<APISuccessResponse<DepositAddressV2[]>> {
+    return this.getPrivate('api/v2/deposit-addresses', params);
+  }
+
+  /**
+   * @deprecated This method is deprecated. Please use getDepositAddressesV3() instead.
+   */
+  getDepositAddressV1(params: {
+    currency: string;
+    chain?: string;
+  }): Promise<APISuccessResponse<DepositAddress>> {
+    return this.getPrivate('api/v1/deposit-addresses', params);
+  }
+
+  /**
+   * @deprecated This method is deprecated.
+   * It is recommended to use the getDeposits() endpoint instead of this endpoint
+   */
+  getHistoricalDepositsV1(
+    params?: GetDepositsRequest,
+  ): Promise<APISuccessResponse<V1HistoricalDeposits>> {
+    return this.getPrivate('api/v1/hist-deposits', params);
+  }
+
+  /**
+   *
+   * REST - FUNDING -Withdrawals
+   * DEPRECATED
+   */
+
+  /**
+   * @deprecated This method is deprecated.
+   * It is recommended to use the getWithdrawals() endpoint instead of this endpoint
+   */
+  getHistoricalWithdrawalsV1(
+    params?: GetWithdrawalsRequest,
+  ): Promise<APISuccessResponse<HistoricalWithdrawalsV1>> {
+    return this.getPrivate('api/v1/hist-withdrawals', params);
+  }
+
+  /**
+   * @deprecated This method is deprecated. Please use submitWithdrawV3() instead.
+   */
+  submitWithdraw(
+    params: ApplyWithdrawRequest,
+  ): Promise<APISuccessResponse<{ withdrawalId: string }>> {
+    return this.postPrivate('api/v1/withdrawals', params);
+  }
+
+  /**
+   *
+   * REST - FUNDING - Transfer
+   * DEPRECATED
+   */
+
+  /**
+   * @deprecated This method is deprecated.
+   * It is recommended to use the submitFlexTransfer() endpoint instead of this endpoint
+   */
+  submitTransferMasterSub(params: submitTransferMasterSubRequest): Promise<
+    APISuccessResponse<{
+      orderId: string;
+    }>
+  > {
+    return this.postPrivate('api/v2/accounts/sub-transfer', params);
+  }
+
+  /**
+   * @deprecated This method is deprecated.
+   * It is recommended to use the submitFlexTransfer() endpoint instead of this endpoint
+   */
+  submitInnerTransfer(params: InnerTransferRequest): Promise<
+    APISuccessResponse<{
+      orderId: string;
+    }>
+  > {
+    return this.postPrivate('api/v2/accounts/inner-transfer', params);
+  }
+
+  /**
+   *
+   * REST - SPOT TRADING - Orders
+   * DEPRECATED
+   */
+
+  /**
+   * @deprecated This method is deprecated.
+   * It is recommended to use the HF trading endpoints instead of this endpoint
```

Comment:
Can you update these style of comments to specify the name of the method to be used instead? It would make the transition easier. Quite a few of these recommending other endpoints.